stable/stolon/values.yaml

```diff
@@ -175,6 +180,10 @@ sentinel:
   #  - name: STSENTINEL_LOG_LEVEL
   #    value: "info"
 
+monitoringPG:
```

Comment:
since it only relates to the keeper, could you move it inside the keeper block?
```
keeper:
  uid_prefix: "keeper"
   ....
   postgresExporter:
     enabled: false
     port: 9187
```